lesson03/assignment/script.js

```diff
@@ -0,0 +1,69 @@
+/* Setting up data as an object */
+const employee = [
+  {
+    "name": "Steve Smith",
+    "jobTitle": "Project Manage",
+    "company": "Front End Dev Co",
+    "experience": "3 years",
+    "school": "UW",
+    "major": "Marketing",
+    "email": "steve@fedc.com",
+    "linkedInUrl": "steve.linkedinprofile.com",
+  },
+  {
+    "name": "Aaron Katz",
+    "jobTitle": "Full Stack Developer",
+    "company": "Web Sites and More",
+    "experience": "5 years",
+    "school": "SU",
+    "major": "Computer Science",
+    "email": "aaronNotMyemail@uw.edu",
+    "linkedInUrl": "aaron.linkedinprofile.com",
+  },
+  {
+    "name": "Kyle Hendricks",
+    "jobTitle": "Starting Pitcher",
+    "company": "Chicago Cubs",
+    "experience": "12 years",
+    "school": "USC",
+    "major": "Pitching",
+    "email": "kyleH@cubs.com",
+    "linkedInUrl": "kyle.linkedinprofile.com",
+  },
+  {
+    "name": "Michael Jordan",
+    "jobTitle": "Point Guard",
+    "company": "Chicago Bulls",
+    "experience": "20 years",
+    "school": "UNC",
+    "major": "Trash Talking",
+    "email": "mJordan@bulls.com",
+    "linkedInUrl": "mJordas.linkedinprofile.com",
+  },
+];
+
+/* Writing function and assigning to the variable "cardHtml" */
+let cardHtml = employee.map(function(info) {
```

Comment:
Your map seems to be working great, glad you were able to get this one figured out